notebooks/geo/BepiColombo_FlybyVisualization.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        row, col = i // 4, i % 4
</pre><p>I think a single command <code>divmod(i, 4)</code>  can return both the row and col.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>